### PR TITLE
[Video] Add disable autoplay for native, more tweaking

### PR DESCRIPTION
**Pull request by haileyok**

### CODE REVIEW

File: src/view/com/util/List.tsx [hunk 1]
```diff
@@ -68,6 +69,7 @@ function ListImpl<ItemT>(
       onBeginDragFromContext?.(e, ctx)
     },
     onEndDrag(e, ctx) {
+      runOnJS(updateActiveViewAsync)()
```
Comment (haileyok) [inline src/view/com/util/List.tsx:72]: swore i added this in another pr, but maybe it got closed or something

File: src/view/com/util/post-embeds/VideoEmbed.tsx [hunk 2]
```diff
@@ -143,38 +136,45 @@ function InnerWrapper({embed}: Props) {
...
         style={[
           {
             position: 'absolute',
             top: 0,
-            bottom: 0,
             left: 0,
             right: 0,
+            bottom: 0,
+            backgroundColor: 'transparent', // If you don't add `backgroundColor` to the styles here,
+            // the play button won't show up on the first render on android 🥴😮‍💨
```
Comment (haileyok) [inline src/view/com/util/post-embeds/VideoEmbed.tsx:161]: wild

File: src/view/com/util/List.tsx (hunk 1 above)
Comment (mozzius) [reply src/view/com/util/List.tsx:72]: How bizarre, maybe it got lost in a merge?

**Review by mozzius — CHANGES_REQUESTED** [review]
Much improved, but disable autoplay isn't working. Videos are still autoplaying in the feed (although it is working when you close fullscreen)

**Comment by haileyok** [comment]
oops!

**Review by mozzius — APPROVED** [review]
LETS GOOOOOO

**Pull request merged**